src/VisualStudio/IntegrationTest/TestUtilities/InProcess/Editor_InProc.cs

```diff
@@ -361,5 +362,93 @@ private static FixAllSuggestedAction GetFixAllSuggestedAction(IEnumerable<Sugges
 
         public void MessageBox(string message)
             => ExecuteOnActiveView(view =>  System.Windows.MessageBox.Show(message));
+
+        public void VerifyDialog(string dialogAutomationId, bool isOpen)
+        {
+            var dialogAutomationElement = FindDialog(dialogAutomationId, isOpen);
+
+            if ((isOpen && dialogAutomationElement == null) ||
+                (!isOpen && dialogAutomationElement != null))
+            {
+                throw new InvalidOperationException($"Expected the {dialogAutomationId} dialog to be {(isOpen ? "open" : "closed")}, but it is not.");
```

Comment:
Err, actually, I'm not sure about that. These are on the VS side, so I'm not sure it really makes sense there.